### DISCUSSION TITLE
Ricerca cliente tramite il campo tax_code o vat_number

### DISCUSSION THREAD

**Discussion by rastellid** [fattureincloud/api#456] [Q&A] [answered]
Ciao a tutti

Attraverso l'utilizzo delle API (https://developers.fattureincloud.it/api-reference/#get-/c/-company_id-/entities/clients) sarebbe possibile ricercare un cliente per mezzo del campo tax_code o vat_number ?

Non posso utilizzare questo endpoint https://developers.fattureincloud.it/api-reference/#get-/c/-company_id-/entities/clients/-client_id- perché non conosco il client_id che in questo caso è obbligatorio

**Comment by valmoz** [comment, marked as answer]
Buongiorno @rastellid 
le consiglio di leggere la seguente guida: [Filter Results](https://developers.fattureincloud.it/docs/basics/additional-capabilities/filter-results)
In ogni caso, le dovrebbe bastare la chiamata List Clients.

Grazie
Mauro

**Reply by rastellid** [reply]
Grazie @valmoz tuttp chiaro
